src/app-layer-htp.c

```diff
@@ -3516,16 +3279,16 @@ static int HTPParserTest01a(void)
     }
 
     htp_tx_t *tx = HTPStateGetTx(htp_state, 0);
-    htp_header_t *h =  htp_table_get_index(tx->request_headers, 0, NULL);
-    if (strcmp(bstr_util_strdup_to_c(h->value), "Victor/1.0")
-        || tx->request_method_number != HTP_M_POST ||
-        tx->request_protocol_number != HTP_PROTOCOL_1_0)
-    {
+    const htp_header_t *h = htp_tx_request_header_index(tx, 0);
+    if (strcmp(bstr_util_strdup_to_c(htp_header_value(h)), "Victor/1.0") ||
+            htp_tx_request_method_number(tx) != HTP_METHOD_POST ||
+            htp_tx_request_protocol_number(tx) != HTP_PROTOCOL_V1_0) {
         printf("expected header value: Victor/1.0 and got %s: and expected"
```

Comment:
Do we still need these as `printf` statements?

Reply:
The point of the PR is to move to rust, not do any functional changes...

Reply:
Fair, was just curious, as sometimes I see them, and wonder if we could suggest their removal. :P